cmd/cpe2cve/cpe2cve.go

```diff
@@ -190,6 +221,21 @@ func init() {
 	flag.Set("logtostderr", "true")
 }
 
+func httpServer(caches map[string]*cvefeed.Cache, cfg config) {
+	router := gin.Default()
+	router.GET("/cpe/:cpe", func(c *gin.Context) {
+		cpe := c.Param("cpe")
+		cves := processSingle(caches, cfg, cpe)
+		c.JSON(http.StatusOK, gin.H{"resp": cves})
+	})
+	var err = router.Run(cfg.HttpBindAt)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Is it a typo? We already using `flog` in this binary, I don't see a point in bringing in `log` too...